packages/augur-ui/src/modules/account/components/account-view.styles.less

```diff
@@ -140,10 +140,33 @@
 
 :root[theme="BETTING"],
 :root[theme="SPORTS"] {
+  .AccountViewWrap {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+
+    > div:first-of-type {
+      .text-24-bold;
+
+      color: var(--color-primary-text);
+      margin: @size-32;
+    }
+  }
+
   .AccountView {
+    margin: @size-8 @size-32;
+    display: flex;
+    flex-flow: column wrap;
+    align-content: space-between;
+    min-height: @size-1400;
```

Comment:
I would prefer to use grid layout for this page, flex is workable but grid is preferred for layouts that are more complex like this one.

Reply:
Updated to grid layout. Please let me know what you think about it. Thanks.